.moai/specs/SPEC-NO-HYBRID-001/acceptance.md

```
@@ -0,0 +1,172 @@
+
+**Then** at minimum 3 files SHALL contain cross-reference to `single-responsibility.md`:
+  - `.claude/rules/moai/development/agent-authoring.md`
+  - `.claude/rules/moai/development/skill-authoring.md`
+  - `.claude/skills/moai-foundation-cc/SKILL.md` or CLAUDE.md
+**And** each cross-reference SHALL be in a relevant section (not random placement)
+
+---
+
+## Edge Cases
+
+### EC-1: Cohesive Multi-Mode (e.g., `/moai db`)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)